src/main/kotlin/com/amazon/opendistroforelasticsearch/indexstatemanagement/models/ManagedIndex.kt

```diff
@@ -117,15 +143,31 @@ data class ManagedIndex(
             }
 
             return ManagedIndex(
-                    id,
-                    version,
-                    index = requireNotNull(index) { "Managed Index index is null" },
-                    jobName = requireNotNull(name) { "Managed Index name is null" },
-                    enabled = enabled,
-                    jobSchedule = requireNotNull(schedule) { "Managed Index schedule is null" },
-                    jobLastUpdateTime = requireNotNull(lastUpdateTime) { "Managed Index Last update time is null" },
-                    jobEnabledTime = enabledTime
+                id,
+                version,
+                index = requireNotNull(index) { "Managed Index index is null" },
+                jobName = requireNotNull(name) { "Managed Index name is null" },
+                enabled = enabled,
+                jobSchedule = requireNotNull(schedule) { "Managed Index schedule is null" },
+                jobLastUpdatedTime = requireNotNull(lastUpdatedTime) { "Managed Index Last updated time is null" },
+                jobEnabledTime = enabledTime,
+                policyName = requireNotNull(policyName) { "Managed index policy name is null" },
```

Comment:
Was keeping convention with alerting since it follows same format and there is no REST API for ManagedIndex as it's an internal job. These would show up as an error in logs instead as this would mean the job sweeper failed to parse this job.